<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-D8G97AUN" ignored="4" name="Test" tests="2" failures="1" timestamp="11 Sep 2020 21:00:48 GMT" time="51.744" errors="0">
  <testcase name="loginLogOut" time="22.088" classname="testcases.US21_1_19_CreateOutgoingDO"/>
  <testcase name="loginLogOut" time="10.849" classname="testcases.US19_AddSubDocumentWeighbridgeOutgoingDO">
    <failure type="java.lang.AssertionError" message="expected [Success] but found [Error]">
      <![CDATA[java.lang.AssertionError: expected [Success] but found [Error]
at pages.ViewWeighBridgePage.validateToastMessage(ViewWeighBridgePage.java:34)
at testcases.US19_AddSubDocumentWeighbridgeOutgoingDO.loginLogOut(US19_AddSubDocumentWeighbridgeOutgoingDO.java:39)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginLogOut -->
  <testcase name="@BeforeTest beforeTest" time="0.0" classname="wrappers.DibizWrappers">
    <ignored/>
  </testcase> <!-- @BeforeTest beforeTest -->
  <testcase name="@AfterSuite afterSuite" time="0.0" classname="wrappers.DibizWrappers">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuite -->
  <testcase name="@BeforeSuite beforeSuite" time="0.0" classname="wrappers.DibizWrappers">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeSuite -->
  <testcase name="@AfterTest afterTest" time="0.0" classname="wrappers.DibizWrappers">
    <ignored/>
  </testcase> <!-- @AfterTest afterTest -->
</testsuite> <!-- Test -->
